CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- Image crate upgraded to 0.20, this is exposed as part of the icon_loading api **breaking**
```

Comment:
```suggestion
- **Breaking:** `image` crate upgraded to 0.20. This is exposed as part of the `icon_loading` API.
```

Thanks for the PR! I just have some tiny nitpicks to make this more consistent with other entries.